lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -135,12 +136,16 @@ def join_bulk_responses(bulk_responses)
     def bulk_send(body_stream)
       params = http_compression ? {:headers => {"Content-Encoding" => "gzip"}} : {}
       # Discard the URL 
+      @metric.increment(:bulk_requests)
+      @metric.increment(:concurrent_bulk_requests)
```

Comment:
For pipelines where the main latency comes from Elasticsearch writes (or the network between), this value is likely to always be the same as number-of-workers. 

For example, if a single batch take 1ms to process and 50ms to land in Elasticsearch (or maybe worse, if you're using Cloud where your ES cluster is very distant). This scenario has each batch taking 51ms, where 98% of the time is spent talking to Elasticsearch, so roughly 98% of the time this `concurrent_bulk_requests` metric will be equal to the number of pipeline workers.

My assumption is that the network is generally slower than the cpu (in terms of time-spent-per-batch), and given this, I do not believe 'concurrent_bulk_requests' to be a useful value because it can be normal and healthy in many cases to have 'concurrent_bulk_requests' any value from 0 to number-of-workers.